lib/macho/load_commands.rb

```diff
@@ -157,16 +157,18 @@ class LoadCommand < MachOStructure
     # @return [Fixnum] the size of the load command, in bytes
     attr_reader :cmdsize
 
-    FORMAT = "VV"
+    FORMAT = "L=2"
     SIZEOF = 8
 
     # Creates a new LoadCommand given an offset and binary string
     # @param offset [Fixnum] the offset to initialize with
     # @param bin [String] the binary string to initialize with
     # @return [MachO::LoadCommand] the new load command
     # @api private
-    def self.new_from_bin(raw_data, offset, bin)
-      self.new(raw_data, offset, *bin.unpack(self::FORMAT))
+    def self.new_from_bin(raw_data, endianness, offset, bin)
```

Comment:
This lacks documentation of the pre-existing `raw_data` argument and the newly added `endianess` argument.